templates/dashboard.php

```diff
@@ -1 +1,7 @@
-<?= sprintf(__('Total count of subscriptions: %d'), $settings['email_count']); ?>
+<?= sprintf(__('Total count of subscriptions: %d'), $settings['email_count']);
+print('<br /><br /><div style="height: 200px; overflow-y: scroll;"><table style="width: 100%;"><thead style="font-weight: bold;"><tr><th>Email address</th><th>Since</th></tr></thead>');
+foreach ($settings['subscribers'] as $subscriber) {
+    printf('<tr><td>%s</td><td><time datetime="%s">%s</time></td></tr>', $subscriber['email'], mysql2date('c', $subscriber['date']), mysql2date(get_option('date_format'), $subscriber['date']));
```

Comment:
Can we also include `time` when email was added?

Reply:
The time is included in the `<time>` tag, but is not printed. You could use `get_option('time_format')` if you'd like to include it.

Reply:
Same on my side, time was missing. Let me have a closer look on this...